examples/plugins/writing-plugins.rst

```diff
@@ -0,0 +1,131 @@
+            "mlflow.model_registry_store":
+                "file-plugin=mlflow_test_plugin:PluginRegistrySqlAlchemyStore",
+        },
+    )
+
+The elements of this ``entry_points`` dictionary specify our various plugins. Note that you
+can choose to implement one or more plugin types in your package, and need not implement them all.
+The type of plugin defined by each entry point & its corresponding reference implementation in
+MLflow are described below. You can work from the reference implementations when writing your own
+plugin:
+
+.. list-table::
+   :widths: 20 20 20 20
+   :header-rows: 1
+
+   * - Description
+     - Entry-point group
+     - Entry-point name and value
+     - Reference Implementation
+   * - Plugins for overriding definitions of tracking APIs like ``mlflow.log_metric``, ``mlflow.start_run`` for a specific
+       tracking URI scheme.
+     - mlflow.tracking_store
+     - The entry point value (e.g. ``mlflow_test_plugin:PluginFileStore``) specifies a custom subclass of
+       `mlflow.tracking.store.AbstractStore <https://github.com/mlflow/mlflow/blob/master/mlflow/store/tracking/abstract_store.py#L8>`_
+       (e.g., the `PluginFileStore class <https://github.com/mlflow/mlflow/blob/master/tests/resources/mlflow-test-plugin/mlflow_test_plugin/__init__.py#L9>`_
+       within the ``mlflow_test_plugin`` module).
+
+       The entry point name (e.g. ``file-plugin``) is the tracking URI scheme with which to associate the custom AbstractStore implementation.
+       In the example above, users who install the plugin & set a tracking URI of the form ``file-plugin://<path>`` will use the custom AbstractStore
+       implementation defined in ``PluginFileStore``. The full tracking URI is passed to the ``PluginFileStore`` constructor.
+     - `FileStore <https://github.com/mlflow/mlflow/blob/master/mlflow/store/tracking/file_store.py>`_
+
+   * - Plugins for defining artifact read/write APIs like ``mlflow.log_artifact``, ``MlflowClient.download_artifacts`` for a specified
+       artifact URI scheme (e.g. the scheme used by your in-house blob storage system).
+     - mlflow.artifact_repository
+     - The entry point value (e.g. ``mlflow_test_plugin:PluginLocalArtifactRepository``) specifies a custom subclass of
+       `mlflow.store.artifact.artifact_repo.ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/artifact_repo.py#L12>`_
+       (e.g., the `PluginLocalArtifactRepository class <https://github.com/mlflow/mlflow/blob/master/tests/resources/mlflow-test-plugin/mlflow_test_plugin/__init__.py#L18>`_
+       within the ``mlflow_test_plugin`` module).
+
+       The entry point name (e.g. ``file-plugin``) is the artifact URI scheme with which to associate the custom ArtifactRepository implementation.
+       In the example above, who install the plugin & log to a run whose artifact URI is of the form "file-plugin://<path>" will use the
```

Comment:
In the example above, who install the plugin & log 
-> In the example above, who install the plugin and log

what do you mean by "who install the plugin"